strikes/common.go

```diff
@@ -32,17 +33,25 @@ func getDBConfig() (string, error) {
 	return "", errors.New("database url must be set in the config file")
 }
 
-func getDBInstanceIdentifier() (string, error) {
+func getHostDBInstanceIdentifier() (string, error) {
 	if viper.IsSet("raids.RDS.aws.config.instance_identifier") {
 		return viper.GetString("raids.RDS.aws.config.instance_identifier"), nil
 	}
 	return "", errors.New("database instance identifier must be set in the config file")
 }
 
+func getHostRDSRegion() (string, error) {
+	if viper.IsSet("raids.RDS.aws.config.region") {
+		return viper.GetString("raids.RDS.aws.config.region"), nil
```

Comment:
This is probably for a later PR, but RDS is specifically Amazon, right? If anything, we'd probably do `raids.aws.rds...` 🤔 Noting this for later discussion.

Reply:
i was more so reading this as raiding a database against different cloud provider. RDS term is aws specific, you are right, maybe we change that to DB or RDMS in a later pr. Also noting that we  need to change the repo name as well